BUILD.gn

```diff
@@ -34,6 +34,18 @@ group("browser_dependencies") {
   ]
 }
 
+action("generate_version") {
```

Comment:
I chose to re-use the already existing generate_version because there was already code for parsing the JSON and generating a similar file called version for CI.  I couldn't use a template file because of the JSON parsing. It had to move up in the file so that it could be referenced.